seeds/endpointType.js

```diff
@@ -129,6 +129,14 @@ const endpointTypes = [
   {
     name: 'FSPIOP_CALLBACK_URL_BULK_TRANSFER_ERROR',
     description: 'Participant callback URL to which bulk transfer error notifications can be sent'
+  },
+  {
+    name: 'FSPIOP_CALLBACK_URL_AUTHORIZATIONS',
+    description: 'Participant callback URL to which authorization requests can be sent'
+  },
+  {
+    name: 'FSPIOP_CALLBACK_URL_TRANSACTION',
+    description: 'Participant callback URL to which transaction requests can be sent'
```

Comment:
@DonChangfoot we have a /transactions resource so may be this will be misleading.. how about something else such as FSPIOP_CALLBACK_URL_TRX_REQ_SERVICE?

Reply:
this will affect the tests as well..

Reply:
@elnyry ok, the `transaction-request-service` currently uses the `FSPIOP_CALLBACK_URL_TRANSACTION` for the transaction requests. Should that be the case or shall I update it to use `FSPIOP_CALLBACK_URL_TRX_REQ_SERVICE`?

Reply:
@DonChangfoot I think FSPIOP_CALLBACK_URL_TRX_REQ_SERVICE makes sense to avoid ambiguity

Reply:
@elnyry Can you please look at https://github.com/mojaloop/central-services-shared/pull/159 then I can make the changes for this repo.

Reply:
@DonChangfoot PR-159 approved..